components/zerotier/zerotier.app.mjs

```diff
@@ -32,6 +32,48 @@ export default {
         }));
       },
     },
+    memberHidden: {
+      type: "boolean",
+      label: "Member Hidden?",
+      description: "Whether or not the member is hidden in the UI",
+      optional: true,
+      default: null,
+    },
+    memberName: {
+      type: "string",
+      label: "Member Name",
+      description: "User defined name of the member",
+      optional: true,
+      default: null,
+    },
+    memberDescription: {
+      type: "string",
+      label: "Member Description",
+      description: "User defined description of the member",
+      optional: true,
+      default: null,
+    },
+    memberAuthorized: {
+      type: "boolean",
+      label: "Member Authorized?",
+      description: "Is the member authorized on the network",
+      optional: true,
+      default: null,
+    },
+    memberActiveBridge: {
+      type: "boolean",
+      label: "Allow Ethernet Bridging?",
+      description: "Allow the member to be a bridge on the network",
+      optional: true,
+      default: null,
+    },
+    memberNoAutoAssignIps: {
+      type: "boolean",
+      label: "Do Not Auto-Assign IPs",
+      description: "Pass `TRUE` to exempt this member from the IP auto assignment pool on a Network",
+      optional: true,
+      default: null,
```

Comment:
@michelle0927 the reason default is specified as `null` is because of how the ZT API works. If you pass any key as `null` it will skip it over and not update the value for that key (this allows the user to ignore those fields they don't want to update and be assured it won't update them).

- If it's passed as a boolean of true or false it will actually update the respective value to the boolean defined.
- If it's passed as `undefined` it returns a 500 error.

As far as I can tell, the only option to send values that you don't want to update is to pass them as `null` - that's why I set it like that.

Reply:
https://docs.zerotier.com/central/v1/#operation/updateNetworkMember